### DISCUSSION TITLE
Duda practica html

### DISCUSSION THREAD

**Discussion by Evelo00** [Launch-X-Latam/MisionFrontEnd#222] [Q&A] [answered]
Disculpen no entendi muy bien, cual se supone que esl a practica html?

**Comment by Anthonyah131** [comment, marked as answer]
![Captura de pantalla 2022-10-18 153310](https://user-images.githubusercontent.com/105742243/196549126-5f93a8ac-5c00-4dd8-b41a-839203469ad1.png)

Ahí bro

**Reply by CGux8** [reply]
> ![Captura de pantalla 2022-10-18 153310](https://user-images.githubusercontent.com/105742243/196549126-5f93a8ac-5c00-4dd8-b41a-839203469ad1.png)
> 
> Ahí bro

gracias 
yo tambien andaba perdido xD
